amivapi/blacklist/emails.py

```diff
@@ -67,8 +69,9 @@ def notify_patch_blacklist(new, old):
     """Send an email to a user if one of his entries was updated."""
     # Checks if the particular update resolved the blacklist entry or just
     # fixes an error, for example changed the reason or price. An entry is
-    # resolved when the end_time is before now.
-    if 'end_time' not in new:
+    # resolved when the end_time is before now. The end_time might also
+    # have been removed, in which case we don't schedule an email either.
+    if 'end_time' not in new or new['end_time'] is None:
```

Comment:
Could technically be reduced to `if new.get('end_time') is None`, but I think this might be more easily understandable, thus now change required.